csharp/src/Google.Protobuf.Test/JsonFormatterTest.cs

```diff
@@ -300,7 +319,8 @@ public void WrapperFormatting_IncludeNull()
         {
             // The actual JSON here is very large because there are lots of fields. Just test a couple of them.
             var message = new TestWellKnownTypes { Int32Field = 10 };
-            var formatter = new JsonFormatter(new JsonFormatter.Settings(true));
+            var formatter = new JsonFormatter(new JsonFormatter.Settings(
```

Comment:
I think we may want to add more parameters in the future. It will be good to make the parameters clear